pkg/api/repository.go

```diff
@@ -0,0 +1,35 @@
+package api
+
+import "github.com/content-services/content-sources-backend/pkg/models"
+
+// Repository holds data received from, or returned by, a repositories API request or response
+type Repository struct {
+	UUID                string `json:"uuid"`
+	Name                string `json:"name"`
+	URL                 string `json:"url"`                                //URL of the remote yum repository
+	DistributionVersion string `json:"distribution_version" example:"7"`   //Version to restrict client usage to
+	DistributionArch    string `json:"distribution_arch" example:"x86_64"` //Architecture to restrict client usage to
+	AccountID           string `json:"account_id"`                         //Account ID of the owner
+	OrgID               string `json:"org_id"`                             //Organization ID of the owner
```

Comment:
we should probably mark UUID, AccountId, & OrgID as readonly, like this:  https://github.com/swaggo/swag/blob/93acd2ad51e848ef3c78b25cb3964745fecde054/testdata/format_dst/web/handler.go#L13